### PR TITLE
UX audit: landing page and navigation

### PR DESCRIPTION
**Pull request by CJ-Rose**

### CODE REVIEW

**Comment by dan13ram** [comment]
@CJ-Rose You can see the build fails locally if you run `pnpm -r build` in the root dir. 

you would have to run `pnpm prepare-sepolia && pnpm codegen` in `packages/subgraph` before you can run `pnpm -r build` in the root dir (just to ensure build passes for the subgraph directory)


Here is a screenshot from vercel 

image


I have a feeling its because of version mismatch between the different packages and corresponding issues with node_modules.. 

Can you run `pnpx syncpack list-mistmatches` on the root directory? It will show you all the package version mismatches. 
You can then run `pnpx syncpack fix-mismatches` to fix it. You may have to make some changes post that to ensure the build succeeds. 

Particularly `next` and some `eslint` package versions differ between `dapp` and `website` which may be causing this issue.

File: .eslintrc.js [hunk 1]
```diff
@@ -50,8 +50,13 @@ module.exports = {
     'react/prop-types': 'off', // disable prop-types
     'react/require-default-props': 'off', // disable require default props
     'react/react-in-jsx-scope': 'off', // react in jsx scope
-    'react/jsx-filename-extension': [2, { extensions: ['.ts', '.tsx'] }],
+    'react/jsx-filename-extension': [2, { extensions: ['.js', '.jsx', '.ts', '.tsx'] }],
     'react/function-component-definition': 'off',
+    "react/no-array-index-key": "off",
+    "react/jsx-props-no-spreading": "off",
+    "no-use-before-define": "off",
+    "import/no-default-export": "off",
```
Comment (CJ-Rose) [inline .eslintrc.js:58]: these rules were added to allow /website to build after Next update from v12 to v14, without requiring major code rewrites.

File: packages/contracts/package.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     "deploy-factory": "hardhat run scripts/deploy-factory.ts --network",
     "deploy-spoils-manager": "hardhat run scripts/deploy-spoils-manager.ts --network",
     "deploy-zap": "hardhat run scripts/deploy-zap.ts --network",
-    "flatten-all": "yarn flatten-factory && yarn flatten-instant && yarn flatten-escrow && yarn flatten-updatable && yarn flatten-spoils && yarn flatten-zap && yarn flatten-dao-zap",
+    "flatten-all": "mkdir -p flat && yarn flatten-factory && yarn flatten-instant && yarn flatten-escrow && yarn flatten-updatable && yarn flatten-spoils && yarn flatten-zap && yarn flatten-dao-zap",
```
Comment (CJ-Rose) [inline packages/contracts/package.json:53]: required to execute `pnpm -r build`, build fails if /flat directory does not yet exist.

**Review by CJ-Rose — COMMENTED** [review]
@dan13ram finally, a successful build 😅

want to point out these 2 changes, as they are the only updates to files outside of `/website` and `/docs`

**Review by dan13ram — APPROVED** [review]

**Pull request merged**